python-scripts/xbox/interface/if_xbdm.py

```diff
@@ -52,8 +52,8 @@ def xbdm_parse_response2(length=None):
     lines = []
     while True:
       line = xbdm_read_line()
-      if line == b'.':
-        return lines
+      if line == b'.': #end of response
```

Comment:
nit: space after `#`. Only case where I directly follow `#` by a comment is for `#FIXME`, but even then I often add spaces (depends on how the code around it is formatted). Other instances too.